### DISCUSSION TITLE
Is there way I can override Renovate update behaviour ?

### DISCUSSION THREAD

**Discussion by Fatin1234567** [renovatebot/renovate#19326] [Request Help] [answered]
Say I am using version 0.8.0 of the foo package. The foo maintainers release the following versions:
0.8.1 (patch)
0.9.0 (minor)
1.0.0 (major)
I want renovate to create one pull request for the major only, is there way to do that?

**Comment by rarkins** [comment]
There's a few ways to do this with `packageRules`. First of all, set `matchPackageNames` to limit it to the package you want. Then, you could use `allowedVersions = >=1.0.0`, or you could set `separateMajorMinor=false` if you forever want to take the latest version and not care about separating minor and major.

**Comment by secustor** [comment, marked as answer]
You probably want to set [`separatemajorminor`](https://docs.renovatebot.com/configuration-options/#separatemajorminor) to `false`
